git-branchless/tests/test_amend.rs

```diff
@@ -412,6 +412,41 @@ fn test_amend_head() -> eyre::Result<()> {
     Ok(())
 }
 
+#[test]
+fn test_amend_head_with_file_with_space() -> eyre::Result<()> {
+    let git = make_git()?;
+
+    if !git.supports_committer_date_is_author_date()? {
+        return Ok(());
+    }
```

Comment:
Oh, this may be leftover from the test I copied to create this one. 🤔

Reply:
I'm assuming this is referring to the commented-out lines below (and that this comment is on the wrong lines now due to a force-push?).

Reply:
No, this is the lines I was intending, but I also think that I was wrong here. I was thinking that I wasn't doing anything w/ committer/author data, but then realized that this is probably intended to ensure that the commit IDs generated by `amend` are deterministic for the tests.